docs/sdkx_android/sdk-configuration.mdx

```diff
@@ -304,6 +304,19 @@ For example
     Helpshift.showConversation(MainActivity.this, config);
 ```
 
+## Initiate Chat on widget load {#initiate-chat}
```

Comment:
for mobile sdk, we should not call it "widget load".
maybe something like "Initiate chat on previous issue resolution"

Reply:
Changing the title to "Initiate new conversation on chat screen load"